### DISCUSSION TITLE
how to exit out of compact mode? (macOS)

### DISCUSSION THREAD

**Discussion by Kimberlyrios1111** [KRTirtho/spotube#1635] [Q&A]
below is an image and im just confused on how to revert back to the other mode 
<img width="1440" alt="Screenshot 2024-06-15 at 3 35 57 AM" src="https://github.com/KRTirtho/spotube/assets/147880405/aac14c74-4455-483a-aee1-f0646144ba1a">

**Comment by romkw** [comment]
Same in AUR version. Found shortcut to enter the settings: 'Ctrl + ,'
https://github.com/KRTirtho/spotube/issues/1646#issuecomment-2189305781

**Reply by Kimberlyrios1111** [reply]
> Same in AUR version. Found shortcut to enter the settings: 'Ctrl + ,' [#1646 (comment)](https://github.com/KRTirtho/spotube/issues/1646#issuecomment-2189305781)

im on mac sadly 😔
